CONTRIBUTING.md

```diff
@@ -9,3 +9,45 @@ Contributions to Student Robotics' Runbook should:
 [code-of conduct]: https://opsmanual.studentrobotics.org/about-the-charity/code-of-conduct
 [license]: ./LICENSE
 [contribution-guide]: docs/contributing.md
+
+## Review Process
+
+We should aim to keep review as lightweight as possible, with a bias towards
+getting content live over making it perfect.
+
+We do however want to review the content which is contributed for two reasons:
+
+* content that is substantially unclear or actually wrong will do more harm than
+  a lack of documentation
+
+* some of the content here covers sensitive topics such as safety measures at
+  events or safeguarding; it is especially important that documentation of these
+  areas is correct & clear, warranting more scrutiny
+
+The following protocol (in line with how GitHub describe these terms) is
```

Comment:
What happens if the majority of a review's comments are non-blocking, but a single point is. How is that reflected to the author?

Perhaps mentioning a standard like https://conventionalcomments.org/ would cover that?

Reply:
That's still blocking and it's be on the reviewer to clarify that. 9c6da4e encourages clarity here.

Conventional comments sounds like an interesting thing to try, however I'm concerned that these guidelines are already quite long! (I do separately wonder if we should have a set of generic guidelines for SR as a whole -- that could more easily be more detailed. Alternatively if we can build up a culture of using conventional comments then that alone may suffice)